src/utils.cpp

```diff
@@ -1337,28 +1337,15 @@ std::optional<std::string> abs_path(const std::string &rel_path)
 
 int64_t min_value(const std::vector<uint8_t> &value, int nvalues)
 {
-  int64_t val, max = 0, retval;
+  int64_t val, max = 0;
   for (int i = 0; i < nvalues; i++)
   {
     val = read_data<int64_t>(value.data() + i * sizeof(int64_t));
     if (val > max)
       max = val;
   }
 
-  /*
```

Comment:
Since we no longer need to compare to the uninitialized value, we can remove this hack.